### ISSUE TITLE
Can't read data from Honeywell Scanner

### ISSUE THREAD

**Issue by tresf** [qzind/tray#288]
**Edit:** The solution is to provide a `usagePage` parameter.  This is an API change... (2.0.6 and greater only)

```js
var device = {
   vendorId: '0x0c2e',
   productId: '0x0b87',
   usagePage: '0x008c', //filter out 0xff8c
   readSize: 61,
   interval: 10, //ms
```

**Before**
```js
qz.hid.claimDevice(device.vendorId, device.productId);
```

**After**
```js
qz.hid.claimDevice(device);
```

--- 

We found a problem while working on #286 with the way Honeywell scanners work with QZ Tray... 

* Honeywell installs two USB devices with the same `VID` / `PID`.
* QZ Tray grabs the first device with may NOT be the HID device.
* QZ Tray can never read data from the Honeywell barcode scanner.

A second feature will be added which allows querying on `UsagePage` parameter from the Honeywell scanner.
* `-116`  Incorrect USB device
* ` 140` The HID device, the one we want

Tested on:
* [X] Windows - Works using PureJavaHIDAPI
* [X] MacOS - Works using HID4Java
* [ ] Linux - Broken

**Comment by tresf** [comment]
We've decided to use `usagePage` for identifying the endpoint of the scanner, but there is a slight disparity on Mac... 

* `4294967180` Incorrect USB device ⚠️ 
* `140` The HID device, the one we want

HID4Java appears to be returning a `Long` instead of an `Integer`.  We could add long support to the library, but this seems like it may be a bug with the way HID4Java returns the `usagePage`.

**Comment by tresf** [comment]
Turns out casting 4294967180 to short will make it `-116`.  Implemented via 8eda5b5.

**Comment by tresf** [comment]
Closed via #289.
